### DISCUSSION TITLE
can't push predicates on concatenate

### DISCUSSION THREAD

**Discussion by mk965** [planetscale/discussion#396] [Q&A]
I used TypeORM to connect to the database in Nest.js, and encountered the following error, how to solve it?

```js
{
  type: 'mysql',
  url: 'mysql://******:******@ap-northeast.connect.psdb.cloud/****?ssl={"rejectUnauthorized":true}',
  entities: ['dist/**/*.entity.js'],
  ssl: {
      rejectUnauthorized: true,
  },
  synchronize: true,
}
```

<img width="873" alt="image" src="https://user-images.githubusercontent.com/30465793/220503753-2b0ad92e-218c-4ad2-b132-9e89a2c97abd.png">

**Comment by jgreet** [comment]
Hi, can you share the query that generates the `can't push predicates` error here or in a [support ticket](https://support.planetscale.com/hc/en-us/requests/new)? Thanks 🙇

**Comment by fooforge** [comment]
Hi @mk965,

I just want to confirm real quick that we were able to reproduce this and that we have merged a fix into the Open Source version of Vitess earlier today (see https://github.com/vitessio/vitess/pull/12543). It will take some time before this gets rolled out on PlanetScale, but once that has happened I'm going to let you know.

**Reply by keu210** [reply]
Just ran into this as well. Glad there's a fix on the way. 

> It will take some time before this gets rolled out on PlanetScale

@fooforge do you know how long this might be? weeks, months?

thanks!

**Reply by fooforge** [reply]
@keu210 Sorry, I have missed the ping as I was out on vacation. No ETA yet, but I have bumped our internal issue on this matter. Once I have heard back, I'm going to update you again.

Update:
Looks like @JejeDurden has found a workaround for while we hash this out: https://github.com/planetscale/discussion/discussions/396#discussioncomment-5551413.

**Reply by DutchJelly** [reply]
@fooforge the vitess PR has been merged. Do you have any ETA on the update in planetscale? 

And the workaround isn't really a workaround. I'm still unable to change my schema because of this bug.

**Comment by JejeDurden** [comment]
I managed to solve it by adding the following code in the **TypeOrmModule** params of _app.module_ and also in the **DataSource** of your config.

```
synchronize: false
```

**Reply by DutchJelly** [reply]
This unfortunately removes the ability to change your database schemas. Not a great workaround for me.
